sample_apps/simple_app_python/example_app.py

```diff
@@ -23,16 +25,34 @@
 # Script defaults
 ATTEST_KEY_FILE                 = 'attest_key_file.bin'
 APP_DATA_DIR                    = './app1_data/'
+PROVISIONING_DIR                = './provisioning'
 EXAMPLE_MEASUREMENT             = 'example_app.measurement'
 APP_OP_TYPES                    = 'cold-init, get-certified, run-app-as-client, run-app-as-server'
 PLATFORM_ATTEST_ENDORSEMENT     = 'platform_attest_endorsement.bin'
 PLATFORM_CERT_FILE              = 'platform_file.bin'
 POLICY_STORE                    = 'store.bin'
 POLICY_HOST                     = 'localhost'
 POLICY_HOST_PORT                = 8123
-SERVER_APP_HOST                 = 'localhost'
+SERVER_APP_HOST                 = '127.0.0.1'
```

Comment:
I needed this to get some SSL certificate verification to pass.

Things don't work if this is defined as 'localhost'.